src/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component, ReactChildren, ReactNode } from "react";
+import { DataAttributes } from "../../helpers/Attributes";
+
+export interface TileProps extends React.AriaAttributes, DataAttributes {
+  /**
+   * A href link address that is passed into the Tile component
+   */
+  link?: string | undefined;
```

Comment:
Just to clarify, if you mark a property as not required (i.e. with the ?), you don't need to specify undefined. (same comment for the other properties)

```suggestion
  link?: string;
```